.github/workflows/periodic-reporter.yaml

```
@@ -0,0 +1,147 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'reporter'
+
+on:
+  schedule:
+    # 2 hours after scheduled periodic and once again in the evening
+    - cron: '0 5,17 * * *'
+  workflow_dispatch:
+
+jobs:
+  report:
+    permissions:
+      issues: 'write'
+
+    runs-on: 'ubuntu-latest'
+
+    steps:
+      - uses: 'actions/github-script@v6'
+        with:
+          script: |-
+                  // label for all issues opened by reporter
+                  const periodicLabel = 'periodic-failure';
+
+                  // check if any reporter opened any issues previously
+                  const prevIssues = await github.paginate(github.rest.issues.listForRepo, {
+                    owner: 'GoogleCloudPlatform',
+                    repo: 'terraform-google-secure-cicd',
+                    state: 'open',
+                    labels: [periodicLabel]
+                  });
+                  // createOrCommentIssue creates a new issue or comments on an existing issue.
+                  const createOrCommentIssue = async function (title, txt) {
+                    if (prevIssues.length < 1) {
+                      console.log('no previous issues found, creating one');
+                      await github.rest.issues.create({
+                        ...context.repo,
+                        title: title,
+                        body: txt,
+                        labels: [periodicLabel]
+                      });
+                      return;
+                    }
+                    console.log(`found previous issue ${prevIssues[0].html_url}, adding comment`);
+                    await github.rest.issues.createComment({
+                      ...context.repo,
+                      issue_number: prevIssues[0].number,
+                      body: txt
+                    });
+                  };
+
+                  // updateAndCloseIssue comments on an existing issue and closes it. No-op if no issue exists.
```

nit for the future: this function operates on all matching issues
```suggestion
                  // updateAndCloseIssues comments on any existing issues and closes them. No-op if no issue exists.
```